test-app/docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+name: marklogic-javaclient-test-app
```

Comment:
Hmm - add a note to this for now saying that it's for optional usage. I'm reluctant to add this without also 1) Updating the CONTRIBUTING guide to recommend Docker usage, and 2) Updating the Jenkinsfile to use Docker as well. In fact - for those reasons, don't include this yet. But getting a JIRA ticket and PR together to take care of those two other things would be a good step forward for this repository.

Reply:
https://progresssoftware.atlassian.net/browse/MLE-17231